src/Build/BackEnd/Components/Scheduler/Scheduler.cs

```diff
@@ -1330,7 +1334,7 @@ private void AssignUnscheduledRequestToNode(SchedulableRequest request, int node
             // and produce more references (more work to do.)  This just verifies we do not attempt to send a traversal to
             // an out-of-proc node because doing so is inefficient and presently will cause the engine to fail on the remote
```

Comment:
NIT: I should have done it previously, but can we please update this comment. It is no longer valid. We now can build traversal projects out of proc.
Also I believe that bellow `VerifyThrow` is not necessary and outdated. Why we should fail build for supported scenario? I'd recommend to use Debug.Assert instead or better just delete it.